app/models/public_body.rb

```diff
@@ -193,6 +193,18 @@ def set_api_key!
     self.api_key = SecureRandom.base64(33)
   end
 
+  def self.find_by_name(name)
```

Comment:
This probably needs to get squashed in to the commit where we drop the columns, as that's the point at which these methods would disappear?